src/tests/dictionaryConcepts/actions/dictionaryConcept.test.js

```diff
@@ -217,6 +218,49 @@ describe('Test suite for dictionary concept actions', () => {
     });
   });
 
+  it('should fetch dictionary concepts by name', () => {
+    moxios.wait(() => {
+      const request = moxios.requests.mostRecent();
+      request.respondWith({
+        status: 200,
+        response: [concepts],
+      });
+    });
+
+    const expectedActions = [
+      { type: IS_FETCHING, payload: true },
+      { type: FETCH_DICTIONARY_CONCEPT, payload: [concepts] },
+      { type: IS_FETCHING, payload: false },
+    ];
+
+    const store = mockStore(mockConceptStore);
+
+    return store.dispatch(fetchConceptsByName('search')).then(() => {
+      expect(store.getActions()).toEqual(expectedActions);
+    });
+  });
+
+  it('should handle the error when search by name fails', () => {
+    moxios.wait(() => {
+      const request = moxios.requests.mostRecent();
+      request.respondWith({
+        status: 404,
+      });
+    });
+
+    const expectedActions = [
+      { type: IS_FETCHING, payload: true },
```

Comment:
Can you explain what you mean by expecting type payload to be true and false?

Reply:
@dkayiwa the implementation reuses a reducer from a past implementation. Taking a look at that reducer it was implemented like so 
`
case IS_FETCHING:
      return {
        ...state,
        loading: action.payload,
      };
`    
With that implementation it means they passed true to control the state loading and therefore render a loader when payload is true or not when payload is false.